source/appModules/securecrt.py

```diff
@@ -2,11 +2,10 @@
 #A part of NonVisual Desktop Access (NVDA)
 #This file is covered by the GNU General Public License.
 #See the file COPYING for more details.
-#Copyright (C) 2010-2012 NV Access Limited
+#Copyright (C) 2010-2016 NV Access Limited

```

Comment:
Added my name in the comments section and not in documentation. Let me know if this is ok.  I didn't add it at first because I didn't know if it was proper. dded it and then removed it before last commit :D. It's back now.